lib/101/custom_widget_learn.dart

```diff
@@ -30,6 +30,21 @@ class CustomWidgetLearn extends StatelessWidget {
             title: title,
             onPressed: () {},
           ),
+          const SizedBox(height: 20),
+          ElevatedButton(
+            style: ElevatedButton.styleFrom(
+                backgroundColor: _ColorsUtility.red,
+                shape: const StadiumBorder()),
```

Comment:
Add comma where applicable to improve readability.

```suggestion
                shape: const StadiumBorder(),
                ),
```